sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_copy_model_async.py

```diff
@@ -33,15 +39,15 @@
 
 class CopyModelSampleAsync(object):
 
-    async def copy_model_async(self):
+    async def copy_model_async(self, custom_model_id):
         from azure.core.credentials import AzureKeyCredential
         from azure.ai.formrecognizer.aio import FormTrainingClient
 
         source_endpoint = os.environ["AZURE_FORM_RECOGNIZER_ENDPOINT"]
         source_key = os.environ["AZURE_FORM_RECOGNIZER_KEY"]
         target_endpoint = os.environ["AZURE_FORM_RECOGNIZER_TARGET_ENDPOINT"]
         target_key = os.environ["AZURE_FORM_RECOGNIZER_TARGET_KEY"]
-        source_model_id = os.environ["AZURE_SOURCE_MODEL_ID"]
+        source_model_id = os.getenv("AZURE_SOURCE_MODEL_ID") or custom_model_id
```

Comment:
nit: i think you can do os.getenv("AZURE_SOURCE_MODEL_ID", custom_model_id)

Reply:
good catch! I was using os.environ before and then missed this when I switched to getenv :)